app/views/articles/show.html.erb

```diff
@@ -13,8 +13,13 @@
 
 <div class="l-article-3col-main editorial">
   <div class="l-main__row">
-    <main class="l-main__cell-full" role="main">
+    <main class="l-main__cell-full" role="main" data-article="true">
```

Comment:
This could be `[data-dough-newsletter-article]` so the association with the Newsletter component is clearer.